Emby.Server.Implementations/StartupOptions.cs

```diff
@@ -1,30 +1,43 @@
-using System;
-using System.Linq;
-
 namespace Emby.Server.Implementations
 {
+    using CommandLine;
+
+    /// <summary>
+    /// Class used by CommandLine package when parsing the command line arguments.
+    /// </summary>
     public class StartupOptions
     {
-        private readonly string[] _options;
+        [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (databases files etc.).")]
+        public string PathData { get; set; }
+
+        [Option('c', "configdir", Required = false, HelpText = "Path to use for config data (user policies and puctures).")]
+        public string PathConfig { get; set; }
+
+        [Option('l', "logdir", Required = false, HelpText = "Path to use for writing log files.")]
+        public string PathLog { get; set; }
+
+
+        [Option("ffmpeg", Required = false, HelpText = "Path to external FFmpeg exe to use in place of built-in.")]
+        public string FFmpeg { get; set; }
+
+        [Option("ffprobe", Required = false, HelpText = "ffmpeg and ffprobe switches must be supplied together.")]
+        public string FFprobe { get; set; }
```

Comment:
same

Reply:
same?  I though the two FF switches where ok..  Change how please?

Reply:
See the comment above.
`FFprobePath`

Reply:
Ok got you